test/py/test_device_get_region_info.py

```diff
@@ -136,8 +136,7 @@ def test_device_get_region_info_small_argsz_caps():
     assert info.argsz == 80
     assert info.flags == (VFIO_REGION_INFO_FLAG_READ |
```

Comment:
can we get a comment here? i.e. there are caps, but they don't fit, so we don't expect the flag to be set.